chemprop/args.py

```diff
@@ -114,6 +114,7 @@ class CommonArgs(Tap):
     no_cache_mol: bool = False
     """
     Whether to not cache the RDKit molecule for each SMILES string to reduce memory usage (cached by default).
+    If SSL_pretrain with large dataset, better to set it to be True
```

Comment:
Hi Jackson, for this one, it really depends on the hardware. For my personal computer, I think once it exceeds 300K rdkit mol objects in memory it will just have the out-of-memory problem in the Mem. For SSL pre-train, normally will have more than 1M data, since no need to label the data.

Reply:
Hmm ok, might be best to just always set this to True automatically for this type of training then.